docs/GraphQL/codeExamples.rst

```diff
@@ -0,0 +1,11 @@
+Code examples
+=======================
+
+We have a provided a few code examples here below. 
```

Comment:
remove "a" so that the text is: 
We have provided a few code examples here below